src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1902,6 +1909,39 @@ private void StartAutoEvoPrediction()
 
         UpdateCompoundLastingTimes(compoundBalanceData, nightBalanceData, nominalStorage,
             specificStorages ?? throw new Exception("Special storages should have been calculated"));
+
+        // Handle process list
```

Comment:
This should be split as its own method. For simplicity it can be called from here as it needs the energy balance info.